compiler/rustc_mir_transform/src/pass_manager.rs

```diff
@@ -42,7 +42,7 @@ fn to_profiler_name(type_name: &'static str) -> &'static str {
 
 // const wrapper for `if let Some((_, tail)) = name.rsplit_once(':') { tail } else { name }`
 const fn c_name(name: &'static str) -> &'static str {
-    // FIXME Simplify the implementation once more `str` methods get const-stable.
+    // FIXME(const-hacks) Simplify the implementation once more `str` methods get const-stable.
```

Comment:
Minor typo (in this file only):
```suggestion
    // FIXME(const-hack): Simplify the implementation once more `str` methods get const-stable.
```